library/src/main/scala/scala/scalajs/js/typedarray/TypedArray.scala

```diff
@@ -65,6 +65,21 @@ trait TypedArray[T, Repr] extends ArrayBufferView with js.Iterable[T] {
  *  Static information that exists for any concrete TypedArray
  */
 @js.native
-trait TypedArrayStatic extends js.Object {
+trait TypedArrayStatic[T, Repr] extends js.Object {
```

Comment:
The commit message should clearly explain why this change is fine for the 0.6.x series, because it's non-obvious. It is clearly a source-breaking change, but it is binary compatible. The additions of methods in the open trait also binary compatible, but only because this is a *native JS trait*.

Ideally we should prevent extending this trait from anywhere but this package, but that's not possible unless we put all the typed array definitions in the same file, and make it `sealed`.

Reply:
Updated the commit message in https://github.com/scala-js/scala-js/pull/3710/commits/11138736c15c11070755c56677c1723cc8b2fb8f